.travis.yml

```diff
@@ -13,9 +13,11 @@ php:
 
 before_install:
   - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
-  - travis_retry composer self-update
+  - composer self-update
```

Comment:
Why did you remove this?

Reply:
Just copy-paste from ORM. Why do we have it here?

Reply:
It's simply _retry if fails, up to 3 times_. Would probably add it there rather than remove it here.
Good for network operations since Travis can be unstable randomly.

Reply:
Sure, gonna do that though.